src/core/errorHandler.js

```diff
@@ -74,15 +88,75 @@ export function withErrorHandling({ name, id } = {}) {
         instanceId = generateHandlerId({ name });
         log.debug(`Generated error handler ID: ${instanceId}`);
       }
+
       return (state) => ({
-        WrappedComponent,
-        errorHandlerId: instanceId,
         error: state.errors[instanceId],
+        instanceId,
       });
     };
 
+    const mapDispatchToProps = (dispatch) => ({ dispatch });
```

Comment:
This should be the default implementation of `mapDispatchToProps`, so passing `undefined` instead might work.